cirq/ops/raw_types.py

```diff
@@ -134,6 +135,37 @@ def on(self, *qubits: Qid) -> 'gate_operation.GateOperation':
                     repr(self)))
         return gate_operation.GateOperation(self, list(qubits))
 
+    def _wrap_in_linear_combination(
+            self,
+            coefficient: Union[complex, float, int]=1
+            ) -> 'linear_combinations.LinearCombinationOfGates':
+        from cirq.ops import linear_combinations
+        return linear_combinations.LinearCombinationOfGates({self: coefficient})
+
+    def __add__(self,
```

Comment:
probably gate doc deserves a note that gates can be added, multiplied by numeric, etc.

Reply:
Done.